pkgs/tools/misc/makebootfat/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "makebootfat-1.4";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/advancemame/${name}.tar.gz";
+    sha256 = "0v0g1xax0y6hmw2x10nfhchp9n7vqyvgc33gcxqax8jdq2pxm1q2";
+  };
+
+  meta = with stdenv.lib;
```

Comment:
Ok, but my understanding is that when sets aren't function arguements the opening bracket is on the next line.
http://nixos.org/nixpkgs/manual/#idm140737319244528